sh/tmuxcomplete

```diff
@@ -27,7 +27,8 @@
 # Words including 2000 lines of history per pane
 #     sh tmuxcomplete -c '-S -2000'
 
-if ! tmux info &> /dev/null; then
```

Comment:
I thought this was a POSIX pattern. What version of which shell is not compatible with it? (It's not clear to me after reading https://github.com/wellle/tmux-complete.vim/issues/63 )